### DISCUSSION TITLE
Getting error while running nussknacker on my local after cloning from github with ./buildServer.sh then ./runServer.sh

### DISCUSSION THREAD

**Discussion by mhussain-1** [TouK/nussknacker#3061] [Q&A] [answered]
<img width="974" alt="Screen Shot 2022-05-16 at 3 17 18 PM" src="https://user-images.githubusercontent.com/99477376/168571624-40b7d955-d2ba-4ea8-b39c-82c03c31d70f.png">

**Comment by arkadius** [comment, marked as answer]
Hi. You need to perform `sbt prepareDev` which build some Nussknacker extensions described in [Contributing guide](https://github.com/TouK/nussknacker/blob/staging/CONTRIBUTING.md#running-full-version-of-designer-from-intellij) if you want run full, build by yourself version from command line.

May I ask why you want to build project from sources? Because there is other, easier option to run Nussknacker on the table. It is just following our [Quickstart](https://nussknacker.io/documentation/quickstart/docker) which uses already built version, provided on docker hub.

**Reply by arkadius** [reply]
After taking a longer look into build script, it looks like this `sbt prepareDev` step should be already done during `./buildServer.sh`. Nevertheless, please try to run it again and let me know if it helped. If it not helped, I'll investigate what other went wrong.

**Reply by mhussain-1** [reply]
Thank you so much @arkadius. It worked.
